numpy/core/src/multiarray/scalartypes.c.src

```diff
@@ -554,32 +548,34 @@ timedeltatype_repr(PyObject *self)
 
     /* The value */
     if (scal->obval == NPY_DATETIME_NAT) {
-        ret = PyUnicode_FromString("numpy.timedelta64('NaT'");
+        val = PyUnicode_FromString("'NaT'");
```

Comment:
This is a (good) behavior change for how `np.timedelta('NaT', 'D')` displays, right?

Reply:
Shouldn't change, looks like
```
In [3]: np.timedelta64('NaT', 'D')                                              
Out[3]: numpy.timedelta64('NaT','D')
```
How would you like it to display?

Reply:
My mistake, I misread the old code as `numpy.timedelta64('NaT')`